frontend/lib/core/state/auth_state.dart

```diff
@@ -6,32 +6,34 @@ import 'package:note_keeper/data/models/models.dart';
 import 'package:note_keeper/data/repositories/respositories_impl.dart';
 
 final _authServiceProvider = StateNotifierProvider<AuthService, AuthState>(
-    (ref) => AuthService(ref.read(Repository.auth)));
+    (ref) => AuthService(ref.read(Repository.auth), ref));
 
 class AuthService extends StateNotifier<AuthState> {
   final AuthRepositoryImpl _authRepository;
+  final Ref ref;
 
   static StateNotifierProvider<AuthService, AuthState> get provider =>
       _authServiceProvider;
 
-  AuthService(this._authRepository)
+  AuthService(this._authRepository, this.ref)
       : super(const AuthState.unauthenticated(isLoading: true)) {
     refresh();
+    state = state.copyWith(isLoading: false);
   }
 
   Future<void> refresh() async {
+    // from master repo
     try {
-      final user = await _authRepository.get();
+      final user = await ref.read(BackendDependency.account).get();
```

Comment:
@mandolf0 Any specific reason to directly call `.get()` from BackendDependency.account provider?

Reply:
My mistake. Change reverted.